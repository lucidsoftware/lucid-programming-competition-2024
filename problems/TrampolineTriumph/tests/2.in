3
0.88 0.76 0.96
3
29 39 8
79 90 9.25
93 102 5
