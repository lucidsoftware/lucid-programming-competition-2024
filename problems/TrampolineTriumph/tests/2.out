2
17.556914
3
10.561205
1
9.562048
